### ISSUE TITLE
Enclosures: Table lacks scope

### ISSUE THREAD

**Issue by NiquiCara** [rubyforgood/abalone#542]
The table header `th` tags need to have a scope defined. `scope="col"` is probably good enough. 

The actions column needs to have a table header defined for it. It could be hidden with `sr-only`.

![enclosures-table-head-no-scope](https://user-images.githubusercontent.com/45127691/102562308-ccd11200-40a4-11eb-8f69-21c6c320e3ae.PNG)

**Comment by metamoni** [comment]
Opened a PR for this
